src/standup/service.ts

```diff
@@ -80,28 +75,37 @@ export class StandupService implements IMessageConsumer {
     });
   }
 
+  private getSearchStart(): moment.Moment {
+    let mu = moment().utc();
+    let hour = mu.startOf('hour');
+    return hour.subtract(24, 'hour');
+  }
+
+  private sendResponse(message: string, response: Response) {
```

Comment:
Should we call this `logAndSendResponse`

Reply:
@gorkem hmm, not sure given that logging is what we kinda always do?